### PR TITLE
API for ocm ranking (still alpha), FIX ghost play count ocm, unlimited ghost stamp bug, give meter reward bug, etc

### PR DESCRIPTION
**Pull request by ghkkk090**
current PR log :
-Fix ghost play count when retiring ocm
-API for ocm ranking (not sure if this already correct)
-Fix unlimited ghost stamp return (hopefully no more of this)
-Fix give meter reward bug if playCount still 0
-Hopefully fix ocm HoF bug 
-Fix duplicate id in carOrder
-Fix OCM HoF wrong shopName

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -141,3 +150,8 @@ https.createServer({key, cert}, muchaApp).listen(PORT_MUCHA, '0.0.0.0', 511, ()
 https.createServer({key, cert}, app).listen(PORT_BNGI, '0.0.0.0', 511, () => {
     console.log(`Game server listening on port ${PORT_BNGI}!`);
 })
+
+// Create the API server
+http.createServer(apiApp).listen(PORT_API, '0.0.0.0', 511, () => {
```
Comment (ry00001) [inline src/index.ts:155]: Port 511 is too low.

**Review by ry00001 — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (ghkkk090) [reply src/index.ts]: sry.. i have no idea about this

File: src/index.ts [hunk 2]
```diff
@@ -30,8 +31,13 @@ const appRouter = Router();
 const PORT_ALLNET = 80;
 const PORT_MUCHA = 10082;
 const PORT_BNGI = 9002;
+const PORT_API = 9003;
```
Comment (ry00001) [inline src/index.ts:34]: poggers

File: src/modules/ghost.ts [hunk 3]
```diff
@@ -110,10 +110,29 @@ export default class GhostModule extends Module {
 				}
 			}
 
-			// Get all of the friend cars for the carId provided
+			// Get all of the challenger car for the carId provided except beated car
+			let checkBeatedCar = await prisma.carStampTarget.findMany({
```
Comment (ry00001) [inline src/modules/ghost.ts:114]: whats this formatting about?

**Review by ry00001 — APPROVED** [review]
looks good

**Pull request merged**